### ISSUE TITLE
Add a link or button to jump to online Repos next to the install from storage button

### ISSUE THREAD

**Issue by furopi** [Fox2Code/FoxMagiskModuleManager#223]
I'm using MMM and Magisk for quite a while now and I love it. Over the time I managed to build up quite a long list of installed modules, when I want to check online Repos for what's been updated and whats been added, I would consider it at least useful to have a way to jump to the online repos section.

One could argue about if this functionality is need and maybe would have not top priority #1, but I think it would be at least rather easy to implement. I'd appreciate feedback or thoughts about this very much. Thanks for creating Fox's MMM, it's awesome!

**Comment by Fox2Code** [comment]
Yeah this is a good idea, I think next to the search icon could be better.
And the button hide when the search bubble is expanded.

For now I have health issues being do much to deal with for now, so I can't work on it until I get better.
Next major update will probably focus on UI, so will probably be there in `0.7.0` when i can work on it.

Thank for the feedback, I really appreciate it.

**Comment by Fox2Code** [comment]
Rectification: It is possible, while the view is generated at runtime and recycled, the `RecyclerView` allow us to go directly to an element via an element index, so we only need to get the element index of the "Online Repo" header and ask the recycler view to make us go there.

**Comment by Fox2Code** [comment]
This is really weird, well just keep it open, I'll investigate when my current mental issues will be fixed.
